recipes/thrust/all/conanfile.py

```diff
@@ -0,0 +1,33 @@
+from conans import ConanFile, tools
+import os
+
+class ThrustConan(ConanFile):
+    name = "thrust"
+    license = "Apache License 2.0"
+    description = ("Thrust is a parallel algorithms library which resembles"
+                   "the C++ Standard Template Library (STL). Thrust's high-level "
+                   "interface greatly enhances programmer productivity while "
+                   "enabling performance portability between GPUs and multicore CPUs."
+                   "Interoperability with established technologies "
+                   "(such as CUDA, TBB, and OpenMP) facilitates "
+                   "integration with existing software.")
+    topics = ("conan", "thrust", "parallel", "header_only")
```

Comment:
```suggestion
    topics = ("parallel", "stl", "header-only")
```

https://github.com/topics/header-only